src/reducers/gameReducer.js

```diff
@@ -0,0 +1,42 @@
+import { DRINK_COFFEE, EAT_SNACK, TAKE_NAP, STUDY } from '../actions/handleSelectionAction';
+import { DECREMENT_COUNTER } from '../actions/decrementCounterAction';
+import { TOGGLE_START } from '../actions/toggleStartAction';
+import { RESTART_STATE } from '../actions/restartStateAction';
+
+const initialState = {
+  coffees: 0,
+  snacks: 0,
+  naps: 0,
+  studies: 0,
+  timerCount: 30,
+  hasStarted: false,
+  actions: [
+    { name: DRINK_COFFEE, text: 'Drink Coffee', stateName: 'coffees' },
+    { name: EAT_SNACK, text: 'Snack', stateName: 'snacks' },
+    { name: TAKE_NAP, text: 'Nap', stateName: 'naps' },
+    { name: STUDY, text: 'Study', stateName: 'studies' },
+  ]
```

Comment:
not sure this is the best place to have these. could just be an array in your mood js.